fluture.js

```diff
@@ -41,7 +41,7 @@
   }
 
   function isFuture(m){
-    return (m instanceof FutureClass) || Boolean(m) && m['@@type'] === TYPEOF_FUTURE;
+    return (m instanceof Future) || Boolean(m) && m['@@type'] === TYPEOF_FUTURE;
```

Comment:
The parens are there because I've often had issues with order of evaluation surrounding `instanceof` checks. I became a bit afraid.
The purpose of the `instanceof` check is to provide a high-performance check for 99% of input to this function, where the right-hand side does not need to be evaluated